srcmedia/scss/_typography.scss

```diff
@@ -135,3 +135,17 @@ h2.tagline {
     }
 }
 
+
+sup {
+    vertical-align: super;
+    font-size: smaller;
```

Comment:
this is cool! love the use of relative sizing.

Reply:
I can't take any credit, I just used the default browser styles that the reset was turning off!